tests/regression/CMakeLists.txt

```diff
@@ -122,6 +122,15 @@ gridtools_add_cartesian_regression_test(horizontal_diffusion_functions SOURCES h
 gridtools_add_layout_transformation_test()
 gridtools_add_boundary_conditions_test()
 
+# temporary solution
+# TODO(anstaf): perftest and full regression test suite for cuda2 backend
```

Comment:
Btw. I would still also like to have a single-binary performance test, that runs each test for N times, reports median and confidence intervals for each test. Ideally, this would allow for much simpler pyutils and simple basic performance check for library-users… I will open an issue about this, maybe we can also schedule this for GT 2.0 :P

Reply:
There is a problem, yes. I will try to come up with the solution. For this PR I would prefer to have this target as a smoke test that `cuda2` works.